app/scripts/directives/nav.js

```diff
@@ -204,6 +204,19 @@ angular.module('openshiftConsole')
     return {
       restrict: 'EA',
       templateUrl: 'views/directives/header/header.html',
+      controller: function (AuthService, HomePagePreference) {
+        AuthService.withUser().then(function() {
```

Comment:
I think this breaks the log out page? I'm not sure we can make this change since it will require every page to have an authenticated user.

Can we just check if `$rootScope.user` is set and not try to do anything if it's not.